### DISCUSSION TITLE
I-Still-Dont-Care-About-Cookies makes pages unclickable

### DISCUSSION THREAD

**Discussion by aiolos01** [OhMyGuus/I-Still-Dont-Care-About-Cookies#11782] [Q&A]
Like the title says. When this extension is enabled sometimes the pages (happens to a lot of pages) display properly but you can't click on them or select text. A refresh fixes this. The same thing happens with the original i don't care about cookies too. I'm using the latest Firefox.

I'm posting here because apparently the issues tab is reserved for reporting URLs only.

**Comment by SongsForTheDEV** [comment]
I had this with the original extension as well. Simply disabling it for the current website was always OK for me, as this happens not very often.

**Comment by erik55** [comment]
Yeah, for some websites the general rules don’t work and break the website. Those are mostly css rules in `src/data/css/common.css`. I don’t know if it is possible to exclude websites from this common rules.

**Reply by krystian3w** [reply]
Theoretically, when the background is almost transparent under the pop-up/modal, it can be added to the generic hiding.

The exception mechanism may require not injecting the entire CSS and rewriting it for a specific solution.

**Comment by krystian3w** [comment]
Recently, the Shoper CMS requires closing their pop-up/bar by clicking ‘reject unnecessary’ and complaining that, for example, the map and social media are not loading, or accepting functional/marketing cookies.
When the modal interface is hidden by CSS and the scroll is repaired by CSS instead of clicking or strong blocking (bundle files with random JS hash with that modal), clicking on anything no longer works.

Except that this problem does not fit the description that it works from the second visit onwards.
